Python/tests/test_runner_number_with_unit.py

```diff
@@ -28,8 +28,9 @@ def test_number_with_unit_recognizer(
     for actual, expected in zip(results, expected_results):
         assert actual.text == expected['Text']
         assert actual.type_name == expected['TypeName']
-        assert actual.start == expected['Start']
```

Comment:
Same

Reply:
Some specs did not have a `Start` key so I had to revert the change where this line was recently added until those keys are present.